src/network/Proxy.ts

```diff
@@ -231,51 +314,56 @@ class ForwardProxy {
       timer_ = timerStart(this.connConnectTime);
     }
     const ingressAddress = networkUtils.buildAddress(ingressHost, ingressPort);
-    let lock = this.connectionLocks.get(ingressAddress);
+    let lock = this.forwardConnectionLocks.get(ingressAddress);
     if (lock == null) {
       lock = new Mutex();
-      this.connectionLocks.set(ingressAddress, lock);
+      this.forwardConnectionLocks.set(ingressAddress, lock);
     }
     const release = await lock.acquire();
     try {
-      await this.establishConnection(nodeId, ingressHost, ingressPort, timer_);
+      await this.establishForwardConnection(
+        nodeId,
+        ingressHost,
+        ingressPort,
+        timer_,
+      );
     } finally {
       if (timer === undefined) {
         timerStop(timer_!);
       }
       release();
-      this.connectionLocks.delete(ingressAddress);
+      this.forwardConnectionLocks.delete(ingressAddress);
     }
   }
 
-  @ready(new networkErrors.ErrorForwardProxyNotRunning(), true)
-  public async closeConnection(
+  @ready(new networkErrors.ErrorProxyNotRunning(), true)
+  public async closeForwardConnection(
     ingressHost: Host,
     ingressPort: Port,
   ): Promise<void> {
```

Comment:
Some methods still have `ingressHost` and `egressHost` etc as parameters. This should be changed to `targetProxyHost` and `targetProxyPort`?

Reply:
Fixed all instances of `ingressHost`/port and `egressHost`/port.